### PR TITLE
feat(StatusBaseInput): disabled mouse area when input is disabled

### PR DESCRIPTION
**Pull request by alexandraB99**
also added view icon

Needed for: https://github.com/status-im/status-desktop/issues/4918

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [x] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [x] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

File: src/assets/img/icons/view.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
...
+<path fill-rule="evenodd" clip-rule="evenodd" d="M14 7C14 9.20914 12.2091 11 10 11C7.79086 11 6 9.20914 6 7C6 4.79086 7.79086 3 10 3C12.2091 3 14 4.79086 14 7ZM12.5 7C12.5 8.38071 11.3807 9.5 10 9.5C8.61929 9.5 7.5 8.38071 7.5 7C7.5 5.61929 8.61929 4.5 10 4.5C11.3807 4.5 12.5 5.61929 12.5 7Z" fill="white"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M0.30102 7.48327C1.33585 9.06707 4.96324 14 10 14C15.0368 14 18.6642 9.06707 19.6991 7.48327C19.8927 7.18685 19.8927 6.81315 19.6991 6.51673C18.6642 4.93293 15.0368 7.33509e-07 10 0C4.96324 2.20166e-07 1.33585 4.93293 0.30102 6.51673C0.107342 6.81315 0.107342 7.18685 0.30102 7.48327ZM10 12.5C7.9828 12.5 6.15324 11.5088 4.62731 10.1978C3.82428 9.50788 3.14469 8.76246 2.61201 8.10516C2.08841 7.45905 2.08841 6.54096 2.61201 5.89485C3.14469 5.23755 3.82428 4.49212 4.62731 3.80219C6.15323 2.49115 7.9828 1.5 10 1.5C12.0173 1.5 13.8469 2.49116 15.3728 3.80219C16.1758 4.49212 16.8554 5.23754 17.3881 5.89484C17.9117 6.54095 17.9117 7.45904 17.3881 8.10516C16.8554 8.76245 16.1758 9.50788 15.3728 10.1978C13.8469 11.5088 12.0173 12.5 10 12.5Z" fill="white"/>
+</svg>
```
Comment (PascalPrecht) [inline src/assets/img/icons/view.svg:4]: Has this been added by accident? How is this related to the change of this commit?

Comment (alexandraB99) [reply src/assets/img/icons/view.svg]: https://github.com/status-im/StatusQ/pull/699
